editors/code/src/util.ts

```diff
@@ -0,0 +1,18 @@
+let ENABLED: boolean = false;
+
+export const log = {
+    debug(message?: any, ...optionalParams: any[]): void {
```

Comment:
Might I propose the following?
We should not depend on the signature of `console.log()` itself, the logging module should be abstracted of that.

Let's use the following:
```suggestion
    debug(...messages: [unknown, ...unknown[]]): void {
```
Or to be more explicit
```suggestion
    debug(message: unknown, ...otherMessages: unknown[]): void {
```
So we are not able to call it with 0 arguments...

Reply:
I wish `unknown` was easier to write than `any` ...

Reply:
That’s just a copy paste from console.log signature

Reply:
I'd change that once we actually stop using console.log, it's better to avoid premature abstraction, which is not tested by an actual usage

Reply:
Yeah, I just wrote about it:
> We should not depend on the signature of console.log() itself, the logging module